liquibase-snowflake/src/test/java/liquibase/datatype/BinaryTypeSnowflakeTest.java

```diff
@@ -0,0 +1,73 @@
+package liquibase.datatype;
+
+import liquibase.database.core.OracleDatabase;
+import liquibase.database.core.SnowflakeDatabase;
+import liquibase.datatype.core.BinaryTypeSnowflake;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.Arrays;
+
+import static liquibase.servicelocator.PrioritizedService.PRIORITY_DATABASE;
+import static org.junit.Assert.*;
+
+public class BinaryTypeSnowflakeTest {
+
+
+    BinaryTypeSnowflake binaryTypeSnowflake;
+    SnowflakeDatabase snowflakeDatabase;
+
+    @Before
+    public void setup() {
+        binaryTypeSnowflake = new BinaryTypeSnowflake();
+        snowflakeDatabase = new SnowflakeDatabase();
+    }
+
+    @Test
+    public void toDatabaseDataType() {
+        DatabaseDataType databaseDataType = binaryTypeSnowflake.toDatabaseDataType(snowflakeDatabase);
+        assertEquals("BINARY", databaseDataType.getType());
+        assertEquals("BINARY", databaseDataType.toSql());
+    }
+
+    @Test
+    public void binaryAliases() {
+        LiquibaseDataType liquibaseDataType = DataTypeFactory.getInstance().fromDescription("binary", snowflakeDatabase);
+        String[] aliases = liquibaseDataType.getAliases();
+        assertEquals(12, aliases.length);
+        assertTrue(Arrays.asList(aliases).contains("binary"));
+        assertTrue(Arrays.asList(aliases).contains("longblob"));
+        assertTrue(Arrays.asList(aliases).contains("longvarbinary"));
+        assertTrue(Arrays.asList(aliases).contains("java.sql.Types.BLOB"));
+        assertTrue(Arrays.asList(aliases).contains("java.sql.Types.LONGVARBINARY"));
+    }
+
+    @Test
+    public void blobConvertsToBinary() {
+        LiquibaseDataType liquibaseDataType = DataTypeFactory.getInstance().fromDescription("binary", snowflakeDatabase);
+        assertEquals("liquibase.datatype.core.BinaryTypeSnowflake", liquibaseDataType.getClass().getName());
+    }
+
+    @Test
+    public void supports() {
+        assertTrue(binaryTypeSnowflake.supports(snowflakeDatabase));
+        assertFalse(binaryTypeSnowflake.supports(new OracleDatabase()));
+    }
+
+    @Test
+    public void getPriority() {
+        assertEquals(PRIORITY_DATABASE, binaryTypeSnowflake.getPriority());
+    }
+
+    @Test
+    public void getMinParameters() {
```

Comment:
It could be useful to add validation for `getMinParameters` and `getMaxParameters` where these values are configured.

CC @rberezen

Reply:
sure update these tests